camunda-formio-tasklist-vue/src/components/TaskList.vue

```diff
@@ -653,8 +654,15 @@ export default class Tasklist extends Mixins(TaskListMixin) {
       this.token,
       processDefinitionId,
       this.bpmApiUrl
+    );  
+   
+    const getActivity = await CamundaRest.getProcesActivity(
+      this.token,
+      this.task.processInstanceId,
```

Comment:
![image](https://user-images.githubusercontent.com/70306694/147644432-c0810257-10c7-44dd-9519-c51de1386376.png)
Type error of TaskPayload is working